src/basinSubModelList.c

```diff
@@ -283,6 +283,32 @@ void loadBasinResources(global_model_parameters *GLOBAL_MODEL_PARAMETERS)
          {
              load_Motu_Bay3_v22p3(GLOBAL_MODEL_PARAMETERS,i);
          }
+         else if (strcmp(GLOBAL_MODEL_PARAMETERS->basin[i],"Motu_Bay2_v22p5") == 0)
+         {
+             load_Motu_Bay2_v22p3(GLOBAL_MODEL_PARAMETERS,i);
+         }
+         else if (strcmp(GLOBAL_MODEL_PARAMETERS->basin[i],"Motu_Bay3_v22p5") == 0)
+         {
+             load_Motu_Bay3_v22p3(GLOBAL_MODEL_PARAMETERS,i);
+         }
+         else if (strcmp(GLOBAL_MODEL_PARAMETERS->basin[i],"Omaio_Bay1_v22p5") == 0)
+         {
+             load_Omaio_Bay1_v22p5(GLOBAL_MODEL_PARAMETERS,i);
+         }
+         else if (strcmp(GLOBAL_MODEL_PARAMETERS->basin[i],"Omaio_Bay2_v22p5") == 0)
+         {
+             load_Omaio_Bay2_v22p5(GLOBAL_MODEL_PARAMETERS,i);
+         }
+         else if (strcmp(GLOBAL_MODEL_PARAMETERS->basin[i],"Omaio_Bay3_v22p5") == 0)
+         {
+             load_Omaio_Bay3_v22p5(GLOBAL_MODEL_PARAMETERS,i);
+         }
+         else if (strcmp(GLOBAL_MODEL_PARAMETERS->basin[i],"TolagaBay_v22p5") == 0)
+         {
+             load_TolagaBay_v22p5(GLOBAL_MODEL_PARAMETERS,i);
```

Comment:
Unfortunately the suggested change I made earlier needs this to be changed to "Tolaga_Bay_v22p5" now.